test/Sentry.AspNetCore.Tests/SentryMiddlewareTests.cs

```diff
@@ -34,6 +34,18 @@ public Fixture()
             Hub.When(hub => hub.ConfigureScope(Arg.Any<Action<Scope>>()))
                 .Do(callback => callback.Arg<Action<Scope>>().Invoke(Scope));
 
+            Hub.When(hub => hub.ConfigureScope(
```

Comment:
Any ideas what the proper way to do this would be? This is the only thing I could get to work, I tried using `Arg.AnyType` in the `Do` implementation but it wasn't able to find the argument.

Reply:
This is maybe as good as it gets...